lib/Dialect/AIEVec/Transforms/VectorToAIEVecConversions.cpp

```diff
@@ -528,8 +492,8 @@ struct ConvertMulAddToAIEVecFMAElemOpPattern
     : public OpConversionPattern<arith::AddIOp> {
   using OpConversionPattern<arith::AddIOp>::OpConversionPattern;
 
-  ConvertMulAddToAIEVecFMAElemOpPattern(MLIRContext *context,
-                                        unsigned shiftParam = 0)
+  explicit ConvertMulAddToAIEVecFMAElemOpPattern(MLIRContext *context,
```

Comment:
Single-argument constructors must be marked explicit to avoid unintentional implicit conversions